scipy/interpolate/_polyint.py

```diff
@@ -703,6 +852,11 @@ def barycentric_interpolate(xi, yi, x, axis=0):
         The y coordinates of the points the polynomial should pass through.
     x : scalar or array_like
         Points to evaluate the interpolator at.
+    der : int or list, optional
+        How many derivatives to extract: `None` for all potentially
```

Comment:
```suggestion
    der : int or list or None, optional
        How many derivatives to extract: ``None`` for all potentially
```

Reply:
Usually we don't put backticks at all for None, but I am not sure we are very consistent here.